lib/load-config.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+require = require('esm')(module); // eslint-disable-line no-global-assign
+const path = require('path');
+const fs = require('fs');
+// ESM seems to break `path` and `fs`, require them first before setting it up..
+const pkgConf = require('pkg-conf');
+
+const loadConfigFile = projectDir => {
+	const loaded = require(path.resolve(projectDir, 'ava.config.js'));
+	const config = (loaded.__esModule) ? loaded.default : loaded;
```

Comment:
Would be good to have a fixture with an `__esModule` export.

Let's ignore the edge case of `export default {__esModule: true, default: {}}` 😉